bugwarrior/services/jira.py

```diff
@@ -33,7 +94,10 @@ class JiraConfig(config.ServiceConfig, prefix='jira'):
     verify_ssl: bool = True
     version: int = 5
 
-    @pydantic.root_validator
+    extra_fields: JiraExtraFields = pydantic.Field(
+        default_factory=lambda: JiraExtraFields(''))
```

Comment:
Note that in the rebase I changed this to `extra_fields: typing.Optional[JiraExtraFields] = None` for consistency with other non-required fields with empty defaults.